lib/inferno/apps/cli/templates/%library_name%.gemspec.tt

```diff
@@ -21,13 +21,7 @@ Gem::Specification.new do |spec|
   spec.metadata['inferno_test_kit'] = 'true'
   # spec.metadata['homepage_uri'] = spec.homepage
   # spec.metadata['source_code_uri'] = 'TODO'
-  spec.files = [
-    Dir['lib/**/*.rb'],
-    Dir['lib/**/*.json'],
-    Dir['config/presets/*.json'],
-    Dir['config/presets/*.json.erb'],
-    'LICENSE'
-  ].flatten
+  spec.files         = `[ -d .git ] && git ls-files -z lib config/presets LICENSE`.split("\x0")
```

Comment:
Aren't the files just getting listed here instead? If I create `my-test-kit/docs` and commit it to git it still doesn't get packaged into the gem.

Reply:
The number of times that necessary files haven't been included in a published gem because they're outside of `lib` is pretty small. The number of times they haven't been included because they're a new file type that hadn't been included before is pretty large.

Reply:
If that's the intention then this works.

We should also mention the existence of `spec.files` here: https://inferno-framework.github.io/docs/distributing-tests.html